example/nginx.ctmpl

```diff
@@ -0,0 +1,64 @@
+# This is an example Nginx configuration template file.
```

Comment:
I could go either way, because this is being used by the start script and not the Dockerfile build. An arbitrary distinction, to be sure. :grinning: